content/en/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli.md

```diff
@@ -284,10 +286,20 @@ New environment "mesh-demo" has been successfully created.
 
 A message indicating that the new environment has been created is shown.
 
+If you chose to create a new k8s cluster, enter a name and press `enter`.
```

Comment:
I think we always want the user to create a new K8sCLsuter name that does not conflict... so maybe this prompt should be: Add a name for your Kubernetes clsuter. This name will be used by the Istio Agents. Enter a name and press 'enter'.